src/HassClient/Model/Targets.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace JoySoftware.HomeAssistant.Model
+{
+    /// <summary>
+    ///     Represents a target for a service call in Home Assistant
+    /// </summary>
+    public record HassTarget()
+    {
+        /// <summary>
+        ///     Zero or more entity id to target with the service call
+        /// </summary>
+        [JsonPropertyName("entity_id")]
+        public IEnumerable<string>? EntityIds { get; init; }
+
```

Comment:
Not sure what the best practice is for  collection prooerties in ummutabke tyoes. Maybe IReadOnlyCollection?

Reply:
Good suggestion.